pyproject.toml

```diff
@@ -35,7 +35,7 @@ indent-style = "space"
 skip-magic-trailing-comma = false
 
 [tool.mypy]
-python_version = "3.12.3"
+python_version = "3.12"
```

Comment:
This was showing a warning in mypy tasks since its not required to have patch version.